src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -980,4 +997,461 @@ TEST_F(EnsembleFlowCustomNodeLoadConfigThenExecuteTest, ReferenceLibraryWithRest
     this->checkResponseForCorrectConfiguration();
 }
 
+static const char* pipelineCustomNodeDifferentOperationsConfig = R"(
+{
+    "model_config_list": [],
+    "custom_node_library_config_list": [
+        {
+            "name": "lib_perform_different_operations",
+            "base_path": "/ovms/bazel-bin/src/lib_node_perform_different_operations.so"
+        }
+    ],
+    "pipeline_config_list": [
+        {
+            "name": "my_pipeline",
+            "inputs": ["pipeline_input", "pipeline_factors"],
+            "nodes": [
+                {
+                    "name": "custom_node",
+                    "library_name": "lib_perform_different_operations",
+                    "type": "custom",
+                    "inputs": [
+                        {"input_numbers": {"node_name": "request",
+                                           "data_item": "pipeline_input"}},
+                        {"op_factors": {"node_name": "request",
+                                           "data_item": "pipeline_factors"}}
+                    ],
+                    "outputs": [
+                        {"data_item": "different_ops_results",
+                         "alias": "custom_node_output"}
+                    ]
+                }
+            ],
+            "outputs": [
+                {"pipeline_output": {"node_name": "custom_node",
+                                     "data_item": "custom_node_output"}
+                }
+            ]
+        }
+    ]
+})";
+
+class EnsembleFlowCustomNodeAndDemultiplexerLoadConfigThenExecuteTest : public EnsembleFlowCustomNodeLoadConfigThenExecuteTest {
+protected:
+    void SetUp() override {
+        EnsembleFlowCustomNodeLoadConfigThenExecuteTest::SetUp();
+        configJsonFilePath = directoryPath + "/ovms_config_file.json";
+    }
+    const std::string differentOpsInputName = "pipeline_input";
+    const std::string differentOpsFactorsName = "pipeline_factors";
+};
+
+enum OPS {
```

Comment:
enum class?

Reply:
I would have to iterate over both OPS and helper int at the same time if I would use enum class.